dev/ScrollBar/ScrollBar.vcxitems

```diff
@@ -18,13 +18,11 @@
       <UseVisualStyle>Latest</UseVisualStyle>
       <Version>RS1</Version>
       <Type>ThemeResources</Type>
-      <UseNonstandardCondtionalXAML>true</UseNonstandardCondtionalXAML>
```

Comment:
Are these only used for these tests ?  Was this introduced to workaround some issues with conditional XAML ?

Reply:
It's only test related.